packaging/rpm/alvr.spec

```diff
@@ -0,0 +1,122 @@
+Name: alvr
+Version: 15.2.1
+Release: 1.0.0
+Summary: Stream VR games from your PC to your headset via Wi-Fi
+License: Proprietary (alvr-org)
+Source: v%{version}.tar.gz
+ExclusiveArch: x86_64
+BuildRequires: alsa-lib-devel cairo-gobject-devel cargo clang-devel ffmpeg-devel gcc gcc-c++ ImageMagick libunwind-devel rust rust-atk-sys-devel rust-cairo-sys-rs-devel rust-gdk-sys-devel rust-glib-sys-devel rust-pango-sys-devel vulkan-headers vulkan-loader-devel
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root
+Requires: ffmpeg rpmfusion-free-release rpmfusion-nonfree-release steam
+Requires(post): policycoreutils
+Requires(postun): policycoreutils
+%global debug_package %{nil} 
+
+%description
+ALVR uses technologies like Asynchronous Timewarp and Fixed Foveated Rendering for a smoother experience. All games that work with an Oculus Rift (s) should work with ALVR.
```

Comment:
That doesn't sound like a good description for someone who doesn't know what ALVR is. (make sure to update the other instance too)